### DISCUSSION TITLE
term so_case.titel can't be found in filters Field Kibana.

### DISCUSSION THREAD

**Discussion by stmarya** [Security-Onion-Solutions/securityonion#11388] [Unsupported Versions] [answered]
I just want try Visualize to filters **so_case.title** on kibana, its can be found in query but not in **Buckets**  for the aggregation i use **Terms**.
is there any settings that i have to reconfigure, or any suggestion for this condition.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/136021057/5f78be88-89fd-4644-b691-c1521aab53c2)

**Comment by weslambert** [comment, marked as answer]
This field is a text field, so you can't perform aggregations with it currently. We might look at making it more usable in the future.

**Reply by stmarya** [reply]
is there anyways to delete all of this 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/136021057/a2a3c5f2-8f9f-4275-ac39-d81d3034ef0f)

**Reply by dougburks** [reply]
If you go into each case and change the Status to `closed`, then they will no longer appear in the default Cases view.
